libs/community/langchain_community/vectorstores/kdbai.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import annotations
+
+import logging
+import uuid
+from typing import Any, Iterable, List, Optional, Tuple
+
+import numpy as np
+
+try:
+    import pandas as pd
```

Comment:
can we import this in the functions that use it, the way we import kdbai_client?

Reply:
I made the changes following your feedback. Thanks.